src/Form/QuestionType/QuestionTypeLongAnswer.php

```diff
@@ -0,0 +1,119 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Form\QuestionType;
+
+use Glpi\Application\View\TemplateRenderer;
+use Glpi\Form\Question;
+
+/**
+ * Long answers are multiple lines inputs used to answer questions with as much details as needed.
+ */
+class QuestionTypeLongAnswer implements QuestionTypeInterface
+{
+    public function renderAdminstrationTemplate(?Question $question): string
+    {
+        $template = <<<TWIG
+            {% import 'components/form/fields_macros.html.twig' as fields %}
+
+            {{ fields.textareaField(
+                'default_value',
+                question is not null ? question.fields.default_value : '',
+                "",
+                {
+                    'placeholder': placeholder,
+                    'enable_richtext': true,
+                    'editor_height': "0",
+                    'rows' : 1,
+                    'init': question is not null ? true : false,
+                    'is_horizontal': false,
+                    'full_width'   : true,
+                    'no_label'     : true,
+                }
+            ) }}
+TWIG;
+
+        $twig = TemplateRenderer::getInstance();
+        return $twig->renderFromStringTemplate($template, [
+            'question'    => $question,
+            'placeholder' => __('Long text'),
+        ]);
+    }
+
+    public function renderEndUserTemplate(Question $question): string
+    {
+        // TODO: handle required
+        $template = <<<TWIG
+            {% import 'components/form/fields_macros.html.twig' as fields %}
+
+            {{ fields.textareaField(
+                "answers[" ~ question.fields.id ~ "]",
+                question.fields.default_value,
+                "",
+                {
+                    'enable_richtext': true,
+                    'editor_height': "0",
+                    'rows' : 1,
+                    'init': question is not null ? true : false,
+                    'is_horizontal': false,
+                    'full_width'   : true,
+                    'no_label'     : true,
+                }
+            ) }}
+TWIG;
+
+        $twig = TemplateRenderer::getInstance();
+        return $twig->renderFromStringTemplate($template, [
+            'question' => $question,
+        ]);
+    }
+
+    public function renderAnswerTemplate($answer): string
+    {
+        $template = <<<TWIG
+            <div class="form-control-plaintext">{{ answer|raw }}</div>
```

Comment:
`raw` is not safe when the value comes from user input.
```suggestion
            <div class="form-control-plaintext">{{ answer|safe_html }}</div>
```